<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="msgBrowser">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>260</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFontComboBox" name="fontComboBox"/>
               </item>
               <item>
                <widget class="QComboBox" name="fontSizeComBox">
                 <property name="currentIndex">
                  <number>4</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>11</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>12</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>13</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>14</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>15</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>16</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>17</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>18</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>19</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>20</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>21</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>22</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="boldBtn">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>加粗</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="italicBtn">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>倾斜</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="underlineBtn">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>下划线</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="colorBtn">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>变颜色</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="saveBtn">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>保存</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearBtn">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>清空</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="msgTxtEdit">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="userTabWidget">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <column>
            <property name="text">
             <string>用户名</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sendBtn">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="userNumLabel">
           <property name="text">
            <string>在线人数：0人</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="exitBtn">
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
